portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DefinitionsSearch.testcase

```diff
@@ -79,4 +88,83 @@ definition {
 		ACDefinitions.noViewAddSearchQueryButton(query = "Test Query 5");
 	}
 
+	@description = "Feature ID: LRAC-13934 | Automation ID: LRAC-14680 | Test Summary: Customized query string will still collect search term"
+	@priority = 4
+	test ViewKeywordSearchedNewStringParameter {
+		task ("Create a new query string parameter") {
+			ACDefinitions.addSearchQuerySuccessfully(query = "query");
+
+			ACDefinitions.goToDefinitionType(definitionType = "Search");
+
+			ACDefinitions.viewSearchQuery(query = "query");
+		}
+
+		task ("Connect DXP to Analytics Cloud") {
+			ACSettings.takeTokenFromAnalyticsCloud();
+
+			TestCase.setUpPortalInstance();
+
+			ACUtils.launchDXP();
+
+			ACUtils.addSiteAndPage();
+
+			var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
```

Comment:
```suggestion
			ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
```

Hey @joshitagaki, I think you don't need to store the macro's return in a variable. You can use the `assignedPropertyName` variable when you need the property name.